app/views/users/specific_user.html.erb

```diff
@@ -0,0 +1 @@
+<h1>Specific User's posts</h1>
```

Comment:
- [ ] Kindly remove this file. If you intend to view a user's posts, the `show.html.erb` already takes care of that.

  Side note: This view file does not match the post controller's action. This should be under the posts view directory. Kindly remove the file anyways

Reply:
I have resolved this issue. Thanks for the feedback